pom.xml

```diff
@@ -1362,6 +1383,20 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>io.confluent</groupId>
+                <artifactId>kafka-protobuf-provider</artifactId>
+                <version>${dep.confluent.version}</version>
```

Comment:
This should also be `provided` due to license but it's needed in `TestKafkaProtobufReads` product tests.
https://github.com/trinodb/trino/pull/14734/files#diff-7cdd855d210d35e291a669e4d8bc34963e693ad07501737f5b0fc317f1ba3c67R25